### PR TITLE
[FIX]: Increase number of pixel difference needed to qualify as non-gesture mode navigation

### PR DESCRIPTION
**Pull request by walmat**
Fixes APP-####

### CODE REVIEW

**Review by skylarbarrera — APPROVED** [review]
sure thing

File: src/helpers/statusBarHelper.ts [hunk 1]
```diff
@@ -34,5 +34,5 @@ export const isUsingButtonNavigation = () => {
   const deviceHeight = Dimensions.get('screen').height;
   const windowHeight = Dimensions.get('window').height;
   const bottomNavBarHeight = deviceHeight - windowHeight;
-  return bottomNavBarHeight > 70;
+  return bottomNavBarHeight > 95;
```
Comment (skylarbarrera) [inline src/helpers/statusBarHelper.ts:37]: any good way to not have this hardcoded? not sure exactly how this is being used

Comment (walmat) [reply src/helpers/statusBarHelper.ts:37]: not that I'm aware. There's probably a better way to check what navigation mode an Android user is using though..

Comment (benisgold) [reply src/helpers/statusBarHelper.ts:37]: is this helper basically guessing whether the android has a soft nav bar or not? wondering if we can just get the nav bar height and return whether it's nonzero

**Review by benisgold — APPROVED** [review]

**Pull request merged**